src/makeCalculator.js

```diff
@@ -37,7 +37,40 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const obj = {
+    result: 0,
+
+    add: (a, b) => {
+      return a + b;
+    },
```

Comment:
Try to refactor:
You methods should get 1 arg number and use this. And you don't really need to return the value

```
add(number) {
  this.result += number;
},